lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const worldStateData = require('warframe-worldstate-data');
+
+/**
+ * Translator to convert in-game names to their localizations
+ */
+class Translator {
+		
+	constructor(lang = 'en_US') {
+		this.data = worldStateData[lang];
+		if(typeof this.data !== "object") {
+			throw new TypeError("data with lang="+lang+" doesn't exists. Please check 'warframe-worldstate-data' dep.");
```

Comment:
I'd also prefer the language to be per-function, as per the discussion in #51, so that a new worldstate object isn't needed for each call. The ideal would be building the translations into each platform, so there'd be a default translation set, but also an object for each supported language.... does that make sense?